tests/system/providers/pinecone/example_pinecone_cohere.py

```diff
@@ -37,15 +36,15 @@
     start_date=datetime(2023, 1, 1),
     catchup=False,
 ) as dag:
-
-    @setup
-    @task
-    def create_index():
-        from airflow.providers.pinecone.hooks.pinecone import PineconeHook
-
-        hook = PineconeHook()
-        hook.create_index(index_name=index_name, dimension=768)
-        time.sleep(60)
+    create_index = CreatePodIndexOperator(
+        task_id="create_index",
+        index_name=index_name,
+        dimension=768,
+        replicas=1,
+        shards=1,
+        pods=1,
+        pod_type="p1.x1",
```

Comment:
are all of these required specs for creation of indexes? If they are not specified do we get failures? If so, we should have default values for them in the operator/hook initialisation.

Reply:
No, I don't think it's required. But this task was added to the new pinecone https://github.com/apache/airflow/pull/37307. I have just reused it and tested it

Reply:
As I read the PR description I see that the system tests are failing due to missing spec. What all are the missing specs that are required? If there are some specs that are required, we should either mark those as required positional args or if they are keyword args they should have default values during the operator initialisation.

https://github.com/apache/airflow/blob/778e8c50b987176b15689bb681ac4c48d7a7805a/airflow/providers/pinecone/operators/pinecone.py#L109

Reply:
`specs` arguments are not compulsory. All the arguments in the `specs` are optional.

Reply:
Previously the create_index did not require the specs for `pinecone<3`.

Reply:
Would the below bare minimum operator usage work?
```
create_index = CreatePodIndexOperator(
        task_id="create_index",
        index_name=index_name,
        dimension=1536
)
```

Reply:
Added a example DAG with bare minimum arguments